ibm/service/power/resource_ibm_pi_network_address_group_member.go

```diff
@@ -0,0 +1,249 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceIBMPINetworkAddressGroupMember() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPINetworkAddressGroupMemberCreate,
+		ReadContext:   resourceIBMPINetworkAddressGroupMemberRead,
+		DeleteContext: resourceIBMPINetworkAddressGroupMemberDelete,
+		Importer:      &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			// Arguments
+			Arg_CloudInstanceID: {
+				Description:  "The GUID of the service instance associated with an account.",
+				ForceNew:     true,
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			Arg_Cidr: {
+				Description:  "The member to add in CIDR format.",
+				ExactlyOneOf: []string{Arg_Cidr, Arg_NetworkAddressGroupMemberID},
+				ForceNew:     true,
+				Optional:     true,
+				Type:         schema.TypeString,
+			},
```

Comment:
These should be switched to be in alphabetical order

Reply:
Updated